js/rpg_objects/Game_Interpreter.js

```diff
@@ -1782,20 +1812,24 @@ Game_Interpreter.requestImages = function(list){
                 });
                 break;
 
-            // Chage Battle Back
+            // Change Battle Back
             case 283:
+                if ($gameParty.inBattle()) {
```

Comment:
These are certainly special behaviors.
They're used entirely only in `Scene_Battle`. (as for parallax, only `Scene_Map`)
In addition, the background and parallax are very large in size as compared with the images such as character, so I decline the load.